package.json

```diff
@@ -26,6 +26,7 @@
     "@kth/server": "^4.1.0",
     "body-parser": "^1.20.2",
     "cookie-parser": "^1.4.6",
+    "csv-parser": "^3.0.0",
```

Comment:
This should not be check-in as its not used by the application itself. We just need to install in locally we run the import script.